Job launching after 0.16 seconds in submission.
Running python job.
Changed into dir /home/okz21/NNC/NN_compression/spearmint/sws_kd
Importing script.py
Running script.main()
Traceback (most recent call last):
  File "/home/okz21/spearmint-install/Spearmint-d9e95208e7a242669e2f76bfc838db79d740e8ab/spearmint/launcher.py", line 244, in launch
    result = python_launcher(job)
  File "/home/okz21/spearmint-install/Spearmint-d9e95208e7a242669e2f76bfc838db79d740e8ab/spearmint/launcher.py", line 328, in python_launcher
    result = module.main(job['id'], params)
  File "/home/okz21/NNC/NN_compression/spearmint/sws_kd/script.py", line 13, in main
    acc, sp = retrain_model(alpha, beta, float(params['tau']), float(params['temp']), int(params['mixtures']), 'SWSModel', 'search', savedir)
  File "../../src/retrain_model.py", line 39, in retrain_model
    output = torch.load("{}{}/{}.out.m".format(model_load_dir, model_file, "fc2"))
  File "/anaconda/envs/py35/lib/python3.5/site-packages/torch/serialization.py", line 265, in load
    f = open(f, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: '/home/okz21/NNC/NN_compression/models/mnist_SWSModel_100_search/fc2.out.m'
Problem executing the function
Job failed in 4.23 seconds.
{'temp': array([11]), 'var': array([50.005]), 'mixtures': array([9]), 'tau': array([0.00050001]), 'mean': array([50000.0005])}
(<class 'FileNotFoundError'>, FileNotFoundError(2, 'No such file or directory'), <traceback object at 0x7fc61b90f888>)
